src/components/FindWallet/WalletFilterPersona.tsx

```diff
@@ -0,0 +1,149 @@
+  setFilters: React.Dispatch<React.SetStateAction<WalletFilter>>
+  selectedPersona: number
+  setSelectedPersona: React.Dispatch<React.SetStateAction<number>>
+  showMobileSidebar: boolean
+}
+
+const WalletFilterPersona = ({
+  resetFilters,
+  setFilters,
+  selectedPersona,
+  setSelectedPersona,
+  showMobileSidebar,
+}: WalletFilterPersonaProps) => {
+  const personas = useWalletPersonas()
+
+  return (
+    <Grid
+      gap={showMobileSidebar ? 2 : 4}
+      autoColumns={{ base: "200px", lg: "minmax(0, 1fr)" }}
+      templateColumns={
+        showMobileSidebar
+          ? "repeat(2, 1fr)"
+          : { base: "200px", lg: "minmax(0, 1fr)" }
+      }
+      mb={showMobileSidebar ? 4 : 2}
+      overflowX="auto"
+    >
+      {personas.map((persona, idx) => {
+        return (
+          <GridItem
+            key={persona.title}
+            gridRow={showMobileSidebar ? "auto" : 1}
+          >
+            <Flex
+              h="100%"
+              direction="column"
+              alignItems="flex-start"
+              padding={2}
+              border="solid"
+              borderColor={getPersonaBorderColor(selectedPersona, idx)}
+              bg="background.highlight"
+              borderRadius="base"
+              cursor="pointer"
+              transition="0.5s all"
+              _hover={{
+                borderColor: "primary.hover",
+              }}
+              role="group"
+              onClick={() => {
+                if (idx === selectedPersona) {
+                  resetFilters()
+                  trackCustomEvent({
+                    eventCategory: "UserPersona",
+                    eventAction: `${persona.title}`,
+                    eventName: `${persona.title} false`,
+                  })
+                } else {
+                  setSelectedPersona(idx)
+                  setFilters(persona.presetFilters)
+                  trackCustomEvent({
+                    eventCategory: "UserPersona",
+                    eventAction: `${persona.title}`,
+                    eventName: `${persona.title} true`,
+                  })
+                }
+              }}
+            >
+              <Flex gap={2} mb={showMobileSidebar ? 0 : 1} px={1.5}>
+                <Box role="radio" aria-label={`${persona.title} filter`}>
+                  <Icon
+                    as={MdCircle}
+                    borderRadius="full"
+                    boxSize={2.5}
+                    my={0}
+                    mx={1}
+                    fill={
+                      selectedPersona === idx
+                        ? "primary.base"
+                        : "rgba(0, 0, 0, 0)"
+                    }
+                    background={
+                      selectedPersona === idx ? "primary.base" : "transparent"
+                    }
+                    outline="1.5px solid"
+                    outlineColor="primary.base"
+                    outlineOffset="3px"
+                    fontSize={8}
```

Comment:
fontSize={8} is 2rem correct? @nloureiro Do you know how often we're trying to use this font size? Not blocking for this, but again noting we have some standard font sizes that would be nice to be able to stick to:

<img width="317" alt="image" src="https://github.com/ethereum/ethereum-org-website/assets/54227730/b8b0941e-3b25-48bf-bd73-63edc4d454bd">